docs/reference/rest_api_communities.md

```diff
@@ -293,12 +293,13 @@ Content-Type: application/json
 | Name     | Type    | Location | Description                                                                                                                                                                                                |
 |----------|---------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `q`      | string  | query    | Search query used to filter results based on [ElasticSearch's query string syntax](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-syntax). |
-| `sort`   | string  | query    | Sort search results (default: newest).                                                                                                                                                                     |
+| `sort`   | string  | query    | Sort search results. Built-in options are `"bestmatch"`, `"newest"`, `"oldest"`, `"updated-desc"`, `"updated-asc"`, `"version"` (default: `"bestmatch"` or `"newest"`).                                                                                                                                                                     |
 | `size`   | integer | query    | Specify number of items in the results page (default: 10).                                                                                                                                                 |
 | `page`   | integer | query    | Specify the page of results.                                                                                                                                                                               |
 | `type`   | string  | query    | Specify community type as one of organization, event, topic or project.                                                                                                                                    |
 | `accept` | string  | header   | - `application/json` (default)<br />- `application/vnd.inveniordm.v1+json`                                                                                                                                 |
 
+Sort options for communities can be configured using the `COMMUNITIES_SORT_OPTIONS` config variable as described in the [search customization](../customize/search) section. Note that `"bestmatch"` is only available as a sort option on requests that provide a query string as a `q` parameter. Otherwise `"bestmatch"` is ignored and the default `"newest"` sort is used.
```

Comment:
The failing tests are because the links have to be to the markdown files. Mkdocs then takes care of converting them to regular files. So we have to replace:

```suggestion
Sort options for communities can be configured using the `COMMUNITIES_SORT_OPTIONS` config variable as described in the [search customization](../customize/search.md) section. Note that `"bestmatch"` is only available as a sort option on requests that provide a query string as a `q` parameter. Otherwise `"bestmatch"` is ignored and the default `"newest"` sort is used.
```

and then we should be fine :crossed_fingers: .

Reply:
Ah, right. Thanks